enterprise/frontend/src/metabase-enterprise/sharing/index.js

```diff
@@ -1,5 +1,11 @@
 import { PLUGIN_DASHBOARD_SUBSCRIPTION_PARAMETERS_SECTION_OVERRIDE } from "metabase/plugins";
+import { hasPremiumFeature } from "metabase-enterprise/settings";
 import MutableParametersSection from "metabase-enterprise/sharing/components/MutableParametersSection";
 
-PLUGIN_DASHBOARD_SUBSCRIPTION_PARAMETERS_SECTION_OVERRIDE.Component =
-  MutableParametersSection;
+if (
+  hasPremiumFeature("advanced_config") ||
+  hasPremiumFeature("advanced_config_legacy")
```

Comment:
still not entirely sure about the `_legacy` flag, or if it needs to be a specific feature flag, or if this should be grandfathered at all.

see: https://metaboat.slack.com/archives/C05EWA11Z1V/p1688997401445349?thread_ts=1688977605.852169&cid=C05EWA11Z1V